pkg/rpcclient/notary/contract.go

```diff
@@ -205,6 +217,10 @@ func (c *Contract) SetNotaryServiceFeePerKeyTransaction(fee int64) (*transaction
 // when transaction ends in HALT state. Notice that this setting can be changed
 // only by the network's committee, so use an appropriate Actor. The transaction
 // is not signed and just returned to the caller.
+//
+// Deprecated: this logic has been moved to native Policy contract since 0.102.1.
```

Comment:
Looks like 0.103 with these changes. It won't be usable at all.

Reply:
Can we just remove it? The rationale for keeping it in RPC client and interops is compatibility with the old versions.